### PR TITLE
Milestone 1: Initialized To do list App

### PR DESCRIPTION
**Pull request by prg-04**
### I have done the following to reach this milestone:

- [ ] Add localStorage to store the tasks
- [ ] Add responsiveness to the app
- [ ] Added a functionality to clear all completed tasks
- [ ] Added a functionality to remove all completed tasks on reload
- [ ] Used ES6 modules for the app functionality
- [ ] Bundled the app using Webpack

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        transform: rotate(360deg) scale(1.05);
+      }
+    }
+  }
+
+  #form {
+    display: flex;
+    width: 100%;
+    height: 3rem;
+    // margin: 0 1rem;
```
Comment (Ibizugbe) [inline src/styles.scss]: Kindly take away every commented block of code from your project as it makes it messy and gives it a less professional look😉.

File: src/List.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class List {
+  constructor(description, completed = false) {
+    this.description = description;
+    this.completed = completed;
+    this.id = Date.now();
+  }
+}
+
+export default List;
```
Comment (Ibizugbe) [inline src/List.js:9]: Nice job creating an array template to accommodate the properties of the different tasks 👍. But just as required in this milestone of the project, you are to create an array of object containing some todo Items the should be rendered dynamically on page load👍.

image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @prg-04 👋👋👋, 

tryagain


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [ ] Linters are properly set up and they are passing👍.
- [ ] Webpack has been used to set up the project👍.
- [ ] GitHub flow has been used👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  document.querySelectorAll('.list-group-item').forEach((elem) => {
+    elem.querySelectorAll('.form-check-input').forEach((item) => {
+      if (item.checked === true) {
+        item.parentNode.parentNode.remove();
+        addList.removeCompleted(item);
+        Store.removeList(item);
+      }
+    });
+  });
+};
```
Comment (codepantha) [inline src/index.js:39]: - [ ] [OPTIONAL] Please note that for this milestone, all that is required is the to-do list structure and not the full blown application. The add, update, remove, and clear-all-completed to-do items features will be added in subsequent milestones. The previous reviewer did not point this out and that's why it is OPTIONAL, but I strongly suggest that you remove those features before you merge 👍

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @prg-04 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**